.github/workflows/tests-integration-reusable.yml

```diff
@@ -40,17 +36,6 @@ jobs:
     runs-on: ${{ inputs.os }} 
     container: us-west2-docker.pkg.dev/shepherd-268822/shepherd2/concourse-resource:latest
     steps:
-    - uses: LouisBrunner/checks-action@v2.0.0
```

Comment:
@joaopapereira, does the removal of this step mean that the integration tests run will no longer be reflected in the check?

Reply:
Given that the tests are now part of the unit tests workflow, there is no need to add extra check tests